src/core/core.scale.js

```diff
@@ -1106,25 +1094,25 @@ class Scale extends Element {
 			tx1 = borderValue + axisHalfWidth;
 			tx2 = me.left + tl;
 		} else if (axis === 'x') {
-			if (position === 'center') {
-				borderValue = alignBorderValue((chartArea.top + chartArea.bottom) / 2);
-			} else if (isObject(position)) {
+			if (isObject(position)) {
```

Comment:
Was this a bug?

Reply:
The issue here was that the TypeScript compiler thought there were times when `borderValue` would be undefined - i.e. if `position` wasn't `'center'` or an object. By changing the statement I made it so that `borderValue` could never be `undefined`